configure.in

```diff
@@ -2537,6 +2537,34 @@ if test "$wxUSE_LIBJPEG" != "no" ; then
     fi
 fi
 
+dnl ------------------------------------------------------------------------
+dnl Check for lzma library
+dnl ------------------------------------------------------------------------
+
+LZMA_LINK=
+if test "$wxUSE_LIBLZMA" = "yes"; then
+    AC_CHECK_LIB(lzma, lzma_code, LZMA_LINK=" -llzma")
+    if test "x$LZMA_LINK" = "x" ; then
+        AC_MSG_ERROR([lzma library not found! Use --with-liblzma=no to disable it])
```

Comment:
I'm not sure about this message: does it imply that `--with-liblzma` is now the default and that it needs to be explicitly disabled if the system doesn't have liblzma? If so, this would be wrong, we want to use liblzma if it's available, but we shouldn't fail if it isn't.

If not, then the message is a bit strange as it means that the user launched configure with `--with-liblzma` option explicitly and so it's not really useful to tell him to use `--without-liblzma` (which is the more usual spelling of `--with-liblzma=no` BTW).

Reply:
No, `--with-liblzma` is not set by default, but if user explicitly set `--with-liblzma` and the configure check failed, what is the point to continue?
Currently `--with-liblzma` and `--with-libjbig` are needed for correct libtiff checks with `--with-libtiff=sys`, because the configure script doesn't know how the system libtiff was compiled.